tools/latency_logging/print_timestamps.py

```diff
@@ -0,0 +1,254 @@
+import argparse
+import json
+#import plotly.figure_factory as ff
+import sys
+from collections import defaultdict
+
+from tools.lib.route import Route
+from tools.lib.logreader import LogReader
+    
+DEMO_ROUTE = "9f583b1d93915c31|2022-04-01--08-55-14"
+
+SERVICES = ['camerad', 'modeld', 'plannerd', 'controlsd', 'boardd']
+
+MSGQ_TO_SERVICE = {
+        'roadCameraState':'camerad',
+        'wideRoadCameraState':'camerad',
+        'modelV2':'modeld',
+        'lateralPlan':'plannerd',
+        'longitudinalPlan':'plannerd',
+        'sendcan':'controlsd',
+        'controlsState':'controlsd'
+        }
+
+SERVICE_TO_DURATIONS = {
+        'camerad':['processingTime'],
+        'modeld':['modelExecutionTime', 'gpuExecutionTime'],
+        'plannerd':["solverExecutionTime"],
+        'controlsd':[],
+        'boardd':[]
+        }
+
+def get_relevant_logs(logreader):
```

Comment:
The assumption is that the relevant logs is a small subset of all logs. Since I need to loop though them 3 times, this improves speed.

I could have used this loop as the first of the 3 loops, but to improve readability I assumed that en extra loop of the small subset was negligible.